dcos_test_utils/test_gce.py

```diff
@@ -0,0 +1,78 @@
+import dcos_test_utils.gce as gce
+import dcos_launch
+import yaml
+import os
+import pytest
+
+
+@pytest.fixture(scope='module')
+def gce_wrapper():
+    credentials_path = os.environ.get('GOOGLE_APPLICATION_CREDENTIALS', os.path.expanduser('~') + '/.gce/credentials.json')
+    return gce.GceWrapper(credentials_path)
+
+
+@pytest.fixture(scope='module')
+def config(gce_wrapper):
+    cwd = os.path.dirname(os.path.realpath(__file__))
+    with open(cwd + '/../dcos_launch/sample_configs/gce-onprem.yaml') as cfile:
```

Comment:
use pkg_resources to read in data and add this to the config mock-outs in `conftest.py` That way both the config tests and the gce launcher tests can use the same mocked config